resources/views/users/elements/menu.blade.php

```diff
@@ -28,7 +28,7 @@
                             <span class="badge">{{ count(Auth::user()->notifications()->where('read_at', null)->get()) }}</span>
                         </a>
                         <div class="dropdown-menu dropdown-menu-right menu-notification" aria-labelledby="navbarDropdown" id="detail-notify">
-                            @foreach (Auth::user()->notifications as $notification)
+                            @foreach (Auth::user()->notifications->where('type', 'App\Notifications\Admin\CensoredOrderNotification') as $notification)
```

Comment:
ngoài type này ra, notification của em có type khác à?
đưa vào config hoặc constant cúa Model

Reply:
sử dụng Auth::user()->notifications thì nó load hết toàn bộ notification. mà em với đạt tạo 2 noti các khác nhau. sai về data sẽ hiển thị lỗi ạ.